lexical-benchmark/input.rs

```diff
@@ -89,6 +89,7 @@ macro_rules! json_data {
 }
 
 /// Generate an array of values as static data
+#[allow(unused_macro_rules)]
```

Comment:
This won't work on older nightly or stable versions without producing an error of an unknown lint. I get no warning with `rustc 1.62.0-nightly (a5ad0d29a 2022-05-12)`, but I do get a warning with `rustc 1.62.0-nightly (6dd68402c 2022-05-11)`.